test/EstateRegistry.js

```diff
@@ -0,0 +1,418 @@
+import assertRevert from './helpers/assertRevert'
+
+const BigNumber = web3.BigNumber
+
+const LANDRegistry = artifacts.require('LANDRegistryTest')
+const EstateRegistry = artifacts.require('EstateRegistry')
+const LANDProxy = artifacts.require('LANDProxy')
+
+const NONE = '0x0000000000000000000000000000000000000000'
+
+require('chai')
+  .use(require('chai-as-promised'))
+  .use(require('chai-bignumber')(BigNumber))
+  .should()
+
+/**
+ * Important:
+ *   The LAND registry uses an encoded version of the coordinates as the tokenId which you can find on LANDRegistry#encodeTokenId but
+ *   you'll see that this file uses tokenIds like `1`, `2`, etc.
+ *   This is because encoding a pair like `(0, 1)` returns `1`, `(0, 2)` returns `2`, and so on.
+ */
+contract('EstateRegistry', accounts => {
+  const [creator, user, anotherUser, yetAnotherUser, hacker] = accounts
+
+  let registry = null
+  let proxy = null
+  let land = null
+  let estate = null
+  const _name = 'Estate'
+  const _symbol = 'EST'
+
+  const creationParams = {
+    gas: 7e6,
+    gasPrice: 1e9,
+    from: creator
+  }
+  const sentByUser = { ...creationParams, from: user }
+  const sentByAnotherUser = { ...creationParams, from: anotherUser }
+  const sentByCreator = { ...creationParams, from: creator }
+
+  const sixX = [0, 0, 1, 1, -3, -4]
+  const sixY = [2, -1, 1, -2, 2, 2]
```

Comment:
We are not using this two vars (`sixX` and `sixY`)

What do you think of adding `"test": "npm run lint && scripts/test.sh"` at `package.json` ?